yarn.lock

```diff
@@ -14038,17 +14037,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"playwright@npm:^1.37.1":
```

Comment:
Did someone not run `yarn install` after updating `package.json` or is Yarn just being clowny?

Reply:
That one's probably my fault from yesterday, sorry!

Reply:
No worries!